src/pages/Main.tsx

```diff
@@ -1,7 +1,58 @@
-import { Button } from "@deskpro/app-sdk";
+/* eslint-disable react-hooks/exhaustive-deps */
+import { TextArea, GetStateResponse } from "@deskpro/app-sdk";
+import { Stack, Button, H1, useDeskproAppClient } from "@deskpro/app-sdk";
+import { useState, useEffect } from "react";
+
+import useDebounce from "../utils/debounce";
 
 export const Main = () => {
-    return (
-        <Button text="Greet" onClick={() => alert("Hello")} />
-    );
+  const { client } = useDeskproAppClient();
+
+  const [text, setText] = useState<string>("");
+  const debouncedText = useDebounce(text, 200);
+
+  useEffect(() => {
+    client?.setState<string>("scratchpad", debouncedText);
+  }, [debouncedText]);
+
+  useEffect(() => {
+    client
+      ?.getState<string>("scratchpad")
```

Comment:
Please change this to use `useInitialisedDeskproAppClient()` too.

Also, we need to use `setUserState()` and `getUserState()` as `setState()` will store these notes globally and we want to store them against the user, see: https://app.shortcut.com/deskpro/story/88865/scratchpad-app-mvp